lib/Sema/TypeCheckNameLookup.cpp

```diff
@@ -1039,8 +1048,7 @@ bool swift::maybeDiagnoseMissingImportForMember(const ValueDecl *decl,
     if (modulesToImport.empty())
       return false;
 
-    diagnoseMissingImportsForMember(decl, modulesToImport, sf, loc);
-    return true;
+    return diagnoseMissingImportsForMember(decl, modulesToImport, sf, loc);
```

Comment:
The doc comment for `maybeDiagnoseMissingImportForMember` needs an update for when the function returns true now that we return false on warnings. I think it would also help to copy-paste that part onto `diagnoseMissingImportsForMember`.